### PR TITLE
fix(wallet): show etld on sign panel

### PR DESCRIPTION
**Pull request by Pavneet-Sing**
Resolves https://github.com/brave/brave-browser/issues/26539

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

[solana-sign-etld.webm](https://user-images.githubusercontent.com/11755381/199949484-e969912b-dccb-4682-9d97-2dea60e722f2.webm)

### CODE REVIEW

File: android/java/org/chromium/chrome/browser/crypto_wallet/fragments/dapps/SignTransactionFragment.java [hunk 1]
```diff
@@ -203,6 +209,11 @@ private void updateSignRequestData() {
         updateActionState(mTxRequestNumber == 0);
         updateAccount(fromAddress);
         updateNetwork(coin);
+        if (originInfo != null && URLUtil.isValidUrl(originInfo.originSpec)) {
```
Comment (sujitacharya2005) [inline android/java/org/chromium/chrome/browser/crypto_wallet/fragments/dapps/SignTransactionFragment.java:212]: **originInfo.originSpec**
is this always not null ?

**Review by sujitacharya2005 — APPROVED** [review]
Looks good

File: android/java/org/chromium/chrome/browser/crypto_wallet/fragments/dapps/SignTransactionFragment.java (hunk 1 above)
Comment (Pavneet-Sing) [reply android/java/org/chromium/chrome/browser/crypto_wallet/fragments/dapps/SignTransactionFragment.java:212]: no: https://github.com/brave/brave-core/blob/5359a4377666706adc7bd31c7610d57d7117cda5/android/java/org/chromium/chrome/browser/crypto_wallet/fragments/dapps/SignTransactionFragment.java#L172

**Comment by srirambv** [comment]
Verification passed on Oppo Reno 5 with Android 12 running 1.47.46 x64 build
- Verified etld is shown on all sign transactions

https://user-images.githubusercontent.com/17010094/200023994-717ea059-40de-4d8c-a2fc-4e06f4a7b030.mp4

**Pull request merged**